docs/azure_container_apps.md

```diff
@@ -0,0 +1,31 @@
+# Deploying on Azure Container Apps
```

Comment:
Can you add the full instructions from the readme here, so that this describes the full process?